### PR TITLE
Updated docs to use top-level request wherever needed

### PR DESCRIPTION
**Pull request by ubdussamad**
Now, all the examples in docs are using top-level request wrapper wherever possible.

### CODE REVIEW

File: docs/user-guide.rst [hunk 1]
```diff
@@ -90,6 +90,7 @@ The :class:`~response.HTTPResponse` object provides
     import urllib3
 
     # Making the request (The request function returns HTTPResponse object)
+    # Here we're using module's request method. See the note in above section for details.
```
Comment (ubdussamad) [inline docs/user-guide.rst:93]: I did this because of shazow's [comment](https://github.com/urllib3/urllib3/issues/2223#issuecomment-842380950)

Comment (sethmlarson) [reply docs/user-guide.rst:93]: If it's mentioned in another section above do we need it duplicated?

**Review by sethmlarson — COMMENTED** [review]
Thanks for this! Exciting to see everything we can do with this helper. Few comments for you:

File: docs/user-guide.rst [hunk 2]
```diff
@@ -445,8 +442,7 @@ to :meth:`~poolmanager.PoolManager.request`:
     # <class "urllib3.response.HTTPResponse">
 
     # This request will take more time to process than timeout.
-    http.request(
-        "GET",
+    urllib3.request("GET",
```
[the remark this answers is not in the document]
Comment (ubdussamad) [reply docs/user-guide.rst:445]: Fixed.

File: docs/user-guide.rst (hunk 1 above)
Comment (ubdussamad) [reply docs/user-guide.rst:93]: Sure, I'll get rid of it. The section is right above it anyways.

**Review by ubdussamad — COMMENTED** [review]
I Will Fix those 2 minor issues in the next commit

**Review by sethmlarson — APPROVED** [review]
LGTM, thank you!

**Pull request merged**